ci/environment-git-tip.yml

```diff
@@ -3,6 +3,8 @@ channels:
   - conda-forge
 dependencies:
   - pip:
+    # TODO: Remove `coiled` install once a new release is out
+    - coiled==1.44.1.dev9
```

Comment:
TODO: Revert being merging

Reply:
1.45 (with the fix for GCP creds) will be on pypi very shortly